libdnf/module/ModulePackageContainer.cpp

```diff
@@ -851,6 +851,24 @@ ModulePackageContainer::getModuleState(const std::string& name)
     }
 }
 
+ModulePackage * ModulePackageContainer::getLatestModule(std::vector<ModulePackage *> modulePackages, bool activeOnly)
+{
+    ModulePackage * latest = nullptr;
+    for (ModulePackage * module: modulePackages) {
+        if (isModuleActive(module->getId()) || !activeOnly) {
```

Comment:
I would recommend to test `!activeOnly` first.
`if (!activeOnly || isModuleActive(module)) {`